src/components/common/TmSessionWelcome.vue

```diff
@@ -79,7 +79,14 @@ export default {
   },
   methods: {
     setState(value) {
-      this.$store.commit(`setSessionModalView`, value)
+      if (navigator.userAgent.includes(`Chrome`)) {
+        this.$store.commit(`setSessionModalView`, value)
+      } else {
+        this.$store.commit(`notifyError`, {
+          title: ``,
+          body: `Please use Chrome or Brave.`
```

Comment:
Opera is also a working browser

Reply:
@faboweb Thanks! I've added that to the message.